Mathlib/Order/Antisymmetrization.lean

```diff
@@ -57,6 +57,8 @@ instance [IsRefl α r] : IsRefl α (AntisymmRel r) where
 
 variable {r}
 
+theorem AntisymmRel.of_eq [IsRefl α r] {a b : α} (h : a = b) : AntisymmRel r a b := h ▸ .rfl
```

Comment:
```suggestion
theorem Eq.antisymmRel [IsRefl α r] {a b : α} (h : a = b) : AntisymmRel r a b := h ▸ .rfl
```
to match `Eq.le`

Reply:
We also have `le_of_eq` though, does that mean we should have both names?

Reply:
We could, yes